sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/SuppressReviveManagerTest.java

```diff
@@ -141,6 +149,21 @@ public Boolean call() throws Exception {
                 });
     }
 
+    private static void waitStateNeverOccurs(
+            SuppressReviveManager suppressReviveManager,
+            SuppressReviveManager.State state) {
+
+        Awaitility.await()
```

Comment:
Thanks.  I was going to write that the wait is needed because of async `TASK_FAILED` handling, but that's not true.  The EventBus in the test posts synchronously so we're good.